plugins/package-managers/bundler/src/main/kotlin/Model.kt

```diff
@@ -0,0 +1,144 @@
+    /** The Gem’s name. */
+    val name: String,
+
+    /** The Gem’s specified version. */
+    val version: SpecVersion,
+
+    /** A long description of this Gem. */
+    val description: String? = null,
+
+    /** A short summary of this Gem’s description. */
+    val summary: String? = null,
+
+    /** The URI of this Gem’s homepage. */
+    val homepage: String? = null,
+
+    /** A list of authors for this Gem. */
+    val authors: List<String> = emptyList(),
+
+    /** The license(s) for the Gem. Each license must be a short name, no more than 64 characters. */
+    val licenses: List<String> = emptyList(),
+
+    /** The list of specified dependencies. */
+    val dependencies: List<SpecDependency> = emptyList()
+)
+
+@Serializable
+internal data class SpecVersion(
+    /** The version string, containing numbers and periods, such as "1.0.0.pre" for a prerelease. */
+    val version: String
+)
+
+@Serializable
+internal data class SpecDependency(
+    /** The name of the dependency. */
+    val name: String,
+
+    /** The type of the dependency (with a leading colon), one of ":development" or ":runtime". */
+    val type: String
+)
+
+/**
+ * Version details for a specific Gem version, see
+ * - https://guides.rubygems.org/rubygems-org-api-v2/
+ * - https://github.com/rubygems/rubygems.org/blob/1f308c8d55403ccc04df407399bcafce87aa5016/app/models/rubygem.rb#L211-L239
+ */
+@Serializable
+internal data class VersionDetails(
+    /** The Gem’s name. */
+    val name: String,
+
+    /** The version string, containing numbers and periods, such as "1.0.0.pre" for a prerelease. */
+    val version: String,
+
+    /** A comma-separated list of authors for this Gem. */
+    val authors: String? = null,
+
+    /** A long description of this Gem. */
+    val description: String? = null,
+
+    /** A short summary of this Gem’s description. */
+    val summary: String? = null,
+
+    /** A synthetic field that contains either the descript or summary, if present. */
+    val info: String? = null,
+
+    /** The license(s) for the Gem. Each license must be a short name, no more than 64 characters. */
+    val licenses: List<String> = emptyList(),
+
+    /** The SHA256 hash of the Gem artifact. */
+    val sha: String? = null,
+
+    /** The download URI of the Gem artifact. */
+    val gemUri: String? = null,
+
+    /** The URI of this Gem’s homepage. */
+    val homepageUri: String? = null,
+
+    /** The URI of this Gem’s source code. */
+    val sourceCodeUri: String? = null,
+
+    /** A map of dependencies per scope. */
+    val dependencies: Map<GemScope, List<GemDependency>> = emptyMap()
+)
+
+@Serializable
+internal enum class GemScope {
+    /** Dependencies required during development. */
+    @SerialName("development")
+    DEVELOPMENT,
+
+    /** Dependencies required at runtime. */
+    @SerialName("runtime")
+    RUNTIME;
+
+    override fun toString() = ":${name.lowercase()}"
+}
+
+@Serializable
+internal data class GemDependency(
```

Comment:
Also `GemScope` and `GemDependency` could be nested inside `VersionDetails` for separation and shorter name.